celery/beat.py

```diff
@@ -281,6 +284,17 @@ def tick(self, event_t=event_t, min=min, heappop=heapq.heappop,
                 return min(verify[0], max_interval)
         return min(adjust(next_time_to_run) or max_interval, max_interval)
 
+    def schedules_equal(self, a, b):
+        if set(a.keys()) != set(b.keys()):
+            return False
+        for name, model in a.items():
+            b_model = b.get(name)
+            if not b_model:
+                return False
+            if model.schedule != b_model.schedule:
```

Comment:
This code looks like it is specific the Django DB models used in django_celery_beat. May not be appropriate to include in celery???